examples/kustomize.jsonnet

```diff
@@ -6,8 +6,12 @@ local kp =
   };
 
 local manifests =
-  { ['00namespace-' + name]: kp.kubePrometheus[name] for name in std.objectFields(kp.kubePrometheus) } +
-  { ['0prometheus-operator-' + name]: kp.prometheusOperator[name] for name in std.objectFields(kp.prometheusOperator) } +
+  { ['setup/0namespace-' + name]: kp.kubePrometheus[name] for name in std.objectFields(kp.kubePrometheus) } +
+  {
+    ['setup/prometheus-operator-' + name]: if name != 'serviceMonitor' then kp.prometheusOperator[name]
+    for name in std.objectFields(kp.prometheusOperator)
+  } +
+  { 'prometheus-operator-serviceMonitor': kp.prometheusOperator.serviceMonitor } +
```

Comment:
Can we make sure that the formatting here is pretty much the same compared to `example.jsonnet`?

Reply:
jsonnet fmt seemed to be doing different things in one file vs. the other, but I'll try to make the two match